XML/Other/ReplaceableTestParameters.xml

```diff
@@ -151,7 +151,11 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>INFINIBAND_INTEL_MPI_SETTINGS_IB</ReplaceThis>
-		<ReplaceWith>-env I_MPI_FABRICS ofi -env SECS_PER_SAMPLE=600</ReplaceWith>
+		<ReplaceWith>-env I_MPI_FABRICS shm:dapl -env I_MPI_DAPL_PROVIDER=ofa-v2-ib0 -env SECS_PER_SAMPLE=600</ReplaceWith>
```

Comment:
Are these single node experiments? If so, please remove shm

Reply:
Wouldn't you want to exercise the SHM path as well?

Reply:
If shm is set, then it will use only the shm path (best performance path), and it wont exercise the IB path.

Reply:
I learned from rdma-core mail chains before. I will fix it.

Reply:
Fixed in 5c9b1e184948b8eaa2fe2b77e38b942b2f0638df